relay/sources/Makefile

```diff
@@ -92,6 +92,7 @@ install: build
 	mkdir -p $(DESTDIR)/opt/rudder/etc/relayd/
 	mkdir -p $(DESTDIR)/opt/rudder/etc/ssl/
 	mkdir -p $(DESTDIR)/opt/rudder/etc/rudder-pkg/
+	mkdir -p $(DESTDIR)/opt/rudder/etc/etc/bash_completion.d/
```

Comment:
Why here? it is then deployed to /etc/bash_completion so should not it be:
 	mkdir -p $(DESTDIR)/etc/etc/bash_completion.d/

Reply:
indeed!